MobileCenterPush/MobileCenterPush/MSPush.h

```diff
@@ -0,0 +1,32 @@
+#import "MSServiceAbstract.h"
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Mobile Center push service.
+ */
+@interface MSPush : MSServiceAbstract
+
+/**
+ * Register for remote notifications
+ */
++ (void)registerForRemoteNotifications;
+
+/**
+ * Callback for succesfull registration with device token
+ *
+ * @param deviceToken The device token for remote notifications
+ */
++ (void)didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken;
```

Comment:
This method is quite long. @elamalani, this method is an exposed method to customers. Is this confirmed?